src/main/java/by/beloboky/employee/NotFoundEmployeeException.java

```diff
@@ -0,0 +1,15 @@
+package by.beloboky.employee;
+
+public class NotFoundEmployeeException extends RuntimeException {
+    /**
+     * Constructs a new runtime exception with the specified detail message.
+     * The cause is not initialized, and may subsequently be initialized by a
+     * call to {@link #initCause}.
+     *
+     * @param message the detail message. The detail message is saved for
+     *                later retrieval by the {@link #getMessage()} method.
+     */
+    public NotFoundEmployeeException(String message) {
```

Comment:
Again - exception name doesn't match with the use case. In your case it is incorrect file/input, _not_ non-existent employee / not founded employee